radar/infrastructure_config/bitbucket.yml

```diff
@@ -0,0 +1,17 @@
+version: 1
```

Comment:
Remove this one. BitBucket is already pending in a separate PR.

Reply:
Can we just merge BitBucket PR in to this instead?

Reply:
@sasjo

Reply:
I think we can merge both into #30 instead.

Reply:
So, we can resolve this as-is?

Reply:
No, delete this one. A bitbucket entry already exists in the `release/3.0.0` branch.

Reply:
OK, will delete